src/qt/locale/bitcoin_it.ts

```diff
@@ -2707,7 +2713,7 @@ Use this functionality with extreme caution.</source>
     </message>
     <message>
         <source>If the custom fee is set to 1000 koinu and the transaction is only 250 bytes, then &quot;per kilobyte&quot; only pays 250 koinu in fee, while &quot;total at least&quot; pays 1000 koinu. For transactions bigger than a kilobyte both pay by kilobyte.</source>
-        <translation type="unfinished"></translation>
+        <translation>Se la commissione di transazione è impostata su 1000 koinu e la transazione é solo 250 bytes, la commissione sará solamente di 250 koinu per kilobyte, mentre il totale sará di almeno 1000 koinu. Per transazioni più grandi di un kilobyte entrambi pagano per kilobyte.</translation>
```

Comment:
There are a couple mistakes in this translation; é should be è, sará should be sarà (this appears twice)

Reply:
Otherwise I couldn't find any other mistakes

Reply:
Fantastic suggestion, which allowed me to catch also the same issue in some previous phrases (some coming from Bitcoin, I believe). This happens when you are an Italian speaker not using an Italian keyboard and you do not pay enough attention.